build/Makefile

```diff
@@ -0,0 +1,181 @@
+# Copyright 2022 The SODA Authors.
```

Comment:
"Copyright 2022 The SODA Authors."  -> "Copyright 2023 The SODA Authors."
Also please handle in "boilerplate.go.txt"

Reply:
done